elasticsearch/src/main/java/com/tcdi/zombodb/postgres/PostgresCountAction.java

```diff
@@ -47,7 +48,7 @@ protected void handleRequest(RestRequest request, RestChannel channel, Client cl
             BytesRestResponse response;
             QueryAndIndexPair query;
 
-            query = PostgresTIDResponseAction.buildJsonQueryFromRequestContent(client, request, !isSelectivityQuery, true, true, true);
+            query = PostgresTIDResponseAction.buildJsonQueryFromRequestContent(client, request, !isSelectivityQuery, true, true, false);
```

Comment:
Don't need last boolean argument to `PostgresTIDResponseAction.buildJsonQueryFromRequestContent()`